### PR TITLE
Multiple votes accumulated on one person

### PR DESCRIPTION
**Pull request by MJJojo97**
See [https://github.com/OpenSlides/OpenSlides/pull/6376](url)

### CODE REVIEW

**Comment by MJJojo97** [comment]
I resolved the problems. Now the workflow should run.

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,8 @@ require (
 	github.com/ory/dockertest/v3 v3.8.1
 )
 
+replace github.com/OpenSlides/openslides-autoupdate-service => github.com/MJJojo97/openslides-autoupdate-service v0.4.1-0.20220303130443-4ba426c27c9c
```
Comment (ostcar) [inline go.mod:13]: Please remove this line and afterwards call `go mod tidy`

**Review by ostcar — CHANGES_REQUESTED** [review]
Everything is ready now. You can remove the replace line. Afterwards, this PR can can merged.

**Comment by MJJojo97** [comment]
Done

**Comment by ostcar** [comment]
The breaking test are my fault. I made some backward incompatible changes in the autoupdate service and did not change the code here.

I will do it as fast as possible

**Comment by ostcar** [comment]
I updated the autoupdate-service in https://github.com/OpenSlides/openslides-vote-service/pull/109 If you merge it into your branch, it should work.

File: go.sum [hunk 2]
```diff
@@ -11,10 +11,13 @@ github.com/Microsoft/go-winio v0.5.2/go.mod h1:WpS1mjBmmwHBEWmogvA2mj8546UReBk4v
 github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 h1:TngWCqHvy9oXAN6lEVMRuU21PR1EtLVZJmdB18Gu3Rw=
 github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5/go.mod h1:lmUJ/7eu/Q8D7ML55dXQrVaamCz2vxCfdQBasLZfHKk=
 github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
+<<<<<<< HEAD
+=======
```
Comment (ostcar) [inline go.sum]: This seems to be a merge conflict. The easiest way to solve it is to remove this file and then call

`go mod tidy`

**Comment by ostcar** [comment]
I merged this. Thank you very much for your work.

File: go.sum (hunk 2 above)
Comment (MJJojo97) [reply go.sum]: Yeah exactly. I made a mistake and are currently on exactly this way to solve it... :D 1 sec...

**Comment by MJJojo97** [comment]
Very nice! :-)

**Pull request merged**